models/staging/synthea/_synthea__models.yml

```diff
@@ -769,13 +761,7 @@ models:
       - name: unique_customers
         data_type: integer
         description: "The number of unique patients enrolled with this Payer during the entire simulation."
-      - name: avg_qols
-        data_type: double precision
-        description: "The average Quality of Life Scores (QOLS) for all patients enrolled with this Payer during the entire simulation."
-      - name: member_months
-        data_type: integer
-        description: "The total number of months that patients were enrolled with this Payer during the simulation and paid monthly premiums (if any)."
-      
```

Comment:
more naming issues from me - in the query these exist but are `qols_avg` and `payer_member_months` (not really sure why i decided to add payer_ there haha, could probably remove)

Reply:
Ahhh I see what you mean